docs/concepts/library-development/lifecycle-hooks.md

```diff
@@ -26,28 +26,25 @@ Lifecycle Hook annotations can be placed on any method inside a step.
 
 Every step has an autowired `hookContext` variable which provides steps with relevant information about what triggered the hook.
 
-| Property              | Description                                                                                                                    |
-|-----------------------|--------------------------------------------------------------------------------------------------------------------------------|
-| `hookContext.library` | The name of the library that contributed the step associated with the step. Value is `null` for hooks not triggered by a step. |
-| `hookContext.step`    | The name of the Library Step that triggered the hook. Value is `null` for hooks not triggered by a step.                       |
+~~--8<-- "snippets/hookContext.md"~~
```

Comment:
I think the ~~ are causing this to not be interpolated. the one in [docs/reference/autowired-variables.md](https://github.com/jenkinsci/templating-engine-plugin/pull/272/files#diff-5511781314f183198c3918e3b8c3e711003ee9d088c526ea81a622b9eaaac610) works

Reply:
good catch - not sure how those `~`'s got there. thanks!